src/frontend/src/pages/HomePage/Home.tsx

```diff
@@ -3,33 +3,23 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Typography } from '@mui/material';
-import OverdueWorkPackageAlerts from './OverdueWorkPackageAlerts';
-import UsefulLinks from './UsefulLinks';
-import WorkPackagesByTimelineStatus from './WorkPackagesByTimelineStatus';
-import UpcomingDeadlines from './UpcomingDeadlines';
-import { useCurrentUser, useSingleUserSettings } from '../../hooks/users.hooks';
-import LoadingIndicator from '../../components/LoadingIndicator';
-import ErrorPage from '../ErrorPage';
-import PageLayout from '../../components/PageLayout';
+import { useCurrentUser } from '../../hooks/users.hooks';
+import { isGuest } from 'shared';
+import GuestHomePage from './GuestHomePage';
+import MemberHomePage from './MemberHomePage';
 
-const Home = () => {
-  const user = useCurrentUser();
-  const { isLoading, isError, error, data: userSettingsData } = useSingleUserSettings(user.userId);
+interface HomeProps {
+  onMemberHomePage: boolean;
+  setOnMemberHomePage: (value: boolean) => void;
+}
 
-  if (isLoading || !userSettingsData) return <LoadingIndicator />;
-  if (isError) return <ErrorPage error={error} message={error.message} />;
+const Home = ({ onMemberHomePage, setOnMemberHomePage }: HomeProps) => {
```

Comment:
these shouldnt be taken in as props. I think you should define a useHistoryState here for those two values, and get rid of it from app authenticated and add a global event listener like we talked about before